app/models/reservation.rb

```diff
@@ -0,0 +1,12 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :event
+  belongs_to :skill
+  belongs_to :reservation_status
+  has_many :driven_pairs, class_name: 'Pair', foreign_key: 'person_1'
+  has_many :navigated_pairs, class_name: 'Pair', foreign_key: 'person_2'
+
+  def pairs
+    driven_pairs + navigated_pairs
```

Comment:
I was thinking of organizations that are doing traditional pairing or maybe even mentoring. This would help users identify their role in previous pairs.

Instead this could be a status or role that the event organizer selects for possible attendees (meaning reservations).